client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -12,7 +12,7 @@
         <button class="btn btn-medium pull-left" ng-if="!sending && item._locked && can_unlock() && item.state !== 'published'" ng-click="unlock()" translate>Unlock</button>
         
         <button class="btn btn-medium pull-left" 
-            ng-show="!sending && !_editable && !item._locked && item.state !== 'published' && item.state !== 'scheduled' && publish_enabled" 
+            ng-show="!sending && !_editable && !item._locked && item.state !== 'published' && item.state !== 'corrected' && item.state !== 'killed' && item.state !== 'scheduled' && publish_enabled" 
```

Comment:
This is getting quite big, maybe we should create a property on the scope for it.